File: app/src/main/java/com/aticosoft/appointments/mobile/business/infrastructure/persistence/datanucleus/plugin/PluginDirectory.java
package com.aticosoft.appointments.mobile.business.infrastructure.persistence.datanucleus.plugin;

import com.rodrigodev.common.file.PathUtils;

/**
 * Created by rodrigo on 14/08/15.
 */
public class PluginDirectory {

    private String path;

    public PluginDirectory(String path) {
        this.path = path;
    }

    public String getPath() {
        return path;
    }

    public String join(String filePath) {
        return PathUtils.join(path, filePath);
    }
}

File: app/src/main/java/com/aticosoft/appointments/mobile/business/infrastructure/persistence/datanucleus/plugin/PluginDirectory.kt
package com.aticosoft.appointments.mobile.business.infrastructure.persistence.datanucleus.plugin

import com.rodrigodev.common.file.pathOf

/**
* Created by Rodrigo Quesada on 08/09/15.
*/
internal class PluginDirectory(val path: String) {

    fun append(filePath: String) = pathOf(path, filePath)
}